### ISSUE TITLE
aten_pe pdu RuntimeError preventing load

### ISSUE THREAD

**Issue by chrisns** [mtdcr/homeassistant-aten-pe#1]
Tried this via HACs and got the same issue
https://github.com/home-assistant/core/issues/85471

```
homeassistant-0 homeassistant 2023-01-08 21:02:42.201 ERROR (MainThread) [aiohttp.server] Error handling request
homeassistant-0 homeassistant Traceback (most recent call last):
homeassistant-0 homeassistant   File "/usr/local/lib/python3.10/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
homeassistant-0 homeassistant     resp = await request_handler(request)
homeassistant-0 homeassistant   File "/usr/local/lib/python3.10/site-packages/aiohttp/web_app.py", line 504, in _handle
homeassistant-0 homeassistant     resp = await handler(request)
homeassistant-0 homeassistant   File "/usr/local/lib/python3.10/site-packages/aiohttp/web_middlewares.py", line 117, in impl
homeassistant-0 homeassistant     return await handler(request)
homeassistant-0 homeassistant   File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 60, in security_filter_middleware
homeassistant-0 homeassistant     return await handler(request)
homeassistant-0 homeassistant   File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 227, in forwarded_middleware
homeassistant-0 homeassistant     return await handler(request)
homeassistant-0 homeassistant   File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 28, in request_context_middleware
homeassistant-0 homeassistant     return await handler(request)
homeassistant-0 homeassistant   File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 81, in ban_middleware
homeassistant-0 homeassistant     return await handler(request)
homeassistant-0 homeassistant   File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 236, in auth_middleware
homeassistant-0 homeassistant     return await handler(request)
homeassistant-0 homeassistant   File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 136, in handle
homeassistant-0 homeassistant     result = await result
homeassistant-0 homeassistant   File "/usr/src/homeassistant/homeassistant/components/config/config_entries.py", line 180, in post
homeassistant-0 homeassistant     return await super().post(request, flow_id)
homeassistant-0 homeassistant   File "/usr/src/homeassistant/homeassistant/components/http/data_validator.py", line 73, in wrapper
homeassistant-0 homeassistant     result = await method(view, request, data, *args, **kwargs)
homeassistant-0 homeassistant   File "/usr/src/homeassistant/homeassistant/helpers/data_entry_flow.py", line 110, in post
homeassistant-0 homeassistant     result = await self._flow_mgr.async_configure(flow_id, data)
homeassistant-0 homeassistant   File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 286, in async_configure
homeassistant-0 homeassistant     result = await self._async_handle_step(
homeassistant-0 homeassistant   File "/usr/src/homeassistant/homeassistant/data_entry_flow.py", line 374, in _async_handle_step
homeassistant-0 homeassistant     result: FlowResult = await getattr(flow, method)(user_input)
homeassistant-0 homeassistant   File "/config/custom_components/aten_pe/config_flow.py", line 60, in async_step_user
homeassistant-0 homeassistant     if await self._test_connection(host, port, user_input):
homeassistant-0 homeassistant   File "/config/custom_components/aten_pe/config_flow.py", line 45, in _test_connection
homeassistant-0 homeassistant     await dev.deviceMAC()
homeassistant-0 homeassistant   File "/config/deps/lib/python3.10/site-packages/atenpdu/__init__.py", line 168, in deviceMAC
homeassistant-0 homeassistant     return str(await self.getAttribute("deviceMAC"))
homeassistant-0 homeassistant   File "/config/deps/lib/python3.10/site-packages/atenpdu/__init__.py", line 161, in getAttribute
homeassistant-0 homeassistant     varbind_table = await self._get([(name, index)])
homeassistant-0 homeassistant   File "/config/deps/lib/python3.10/site-packages/atenpdu/__init__.py", line 128, in _get
homeassistant-0 homeassistant     err_indication, err_status, _, varbind_table = await getCmd(
homeassistant-0 homeassistant RuntimeError: await wasn't used with future
```

**Comment by mtdcr** [comment]
The problem is caused by Home Assistant's move to pysnmplib while atenpdu is still using pysnmp. Installing pysnmp also seems to break other snmp-based sensors in Home Assistant.

**Comment by mtdcr** [comment]
Or rather the other way around, like here: https://github.com/home-assistant/core/issues/84593
Can you try downgrading pysnmplib to 5.0.19?

**Comment by chrisns** [comment]
confirm that does resolve the issue, though not a long term answer :(

**Comment by mtdcr** [comment]
Should be fixed with 0.5.0 by porting atenpdu to the version of pysnmplib Home Assistant uses now (>=5.0.20). Please reopen this issue otherwise.

Currently I have no plans to fix this in Home Assistant. It may be as easy as bumping the version of atenpdu to 0.5.0, but I won't be able to perform runtime tests there soon.
